types/mui-datatables/index.d.ts

```diff
@@ -0,0 +1,173 @@
+    filter?: boolean;
+    filterList?: string[];
+    filterOptions?: string[];
+    sort?: boolean;
+    sortDirection?: 'asc' | 'desc';
+    download?: boolean;
+    hint?: string;
+    customHeadRender?: (columnMeta: MUIDataTableCustomHeadRenderer, updateDirection: (params: any) => any) => string;
+    customBodyRender?: (value: any, tableMeta: MUIDataTableMeta, updateValue: (s: any, c: any, p: any) => any) => string | React.ReactNode;
+    setCellProps?: (cellValue: string, rowIndex: number, columnIndex: number) => string;
+}
+
+export interface MUIDataTableOptions {
+    page?: number;
+    count?: number;
+    serverSide?: boolean;
+    rowsSelected?: any[];
+    filterType?: 'dropdown' | 'checkbox' | 'multiselect' | 'textField';
+    textLabels?: MUIDataTableTextLabels;
+    pagination?: boolean;
+    selectableRows?: boolean;
+    IsRowSelectable?: (dataIndex: any) => boolean;
+    resizableColumns?: boolean;
+    expandableRows?: boolean;
+    renderExpandableRow?: (rowData: string[], rowMeta: { dataIndex: number; rowIndex: number }) => React.ReactNode;
+    customToolbar?: () => React.ReactNode;
+    customToolbarSelect?: () => React.ReactNode;
+    customFooter?: () => React.ReactNode;
+    customSort?: (data: any[], colIndex: number, order: string) => any[];
+    elevation?: number;
+    caseSensitive?: boolean;
+    responsive?: 'stacked' | 'scroll';
+    rowsPerPage?: number;
+    rowsPerPageOptions?: number[];
+    rowHover?: boolean;
+    fixedHeader?: boolean;
+    sortFilterList?: boolean;
+    sort?: boolean;
+    filter?: boolean;
+    search?: boolean;
+    print?: boolean;
+    download?: boolean;
+    downloadOptions?: { filename: string; separator: string };
+    viewColumns?: boolean;
+    onRowsSelect?: (currentRowsSelected: any[], rowsSelected: any[]) => void;
+    onRowsDelete?: (rowsDeleted: any[]) => void;
+    onRowClick?: (rowData: string[], rowMeta: { dataIndex: number; rowIndex: number }) => void;
+    onCellClick?: (colIndex: number, rowIndex: number) => void;
+    onChangePage?: (currentPage: number) => void;
+    onChangeRowsPerPage?: (numberOfRows: number) => void;
+    onSearchChange?: (searchText: string) => void;
+    onFilterChange?: (changedColumn: string, filterList: any[]) => void;
+    onColumnSortChange?: (changedColumn: string, direction: string) => void;
+    onColumnViewChange?: (changedColumn: string, action: string) => void;
+    onTableChange?: (action: string, tableState: object) => void;
+    setRowProps?: (row: any[], rowIndex: number) => any;
+}
+
+export type MUIDataTableColumnDef = string | MUIDataTableColumn;
+
+export interface MuiDatatablesTableState {
+    page: number;
+    rowsPerPage: number;
+    filterList: any[];
+}
+
+export interface MUIDataTableProps {
+    title: string;
+    columns: MUIDataTableColumnDef[];
+    data: any[];
+    options?: MUIDataTableOptions;
+}
+
+export const MUIDataTable: React.ComponentType<MUIDataTableProps>;
```

Comment:
This export makes it seem like I should be able to do:
`import { MUIDataTable } from 'mui-datatables';`

But I don't see this export in the library. If it is, then adding it to the test would be great.